src/components/calcite-checkbox/calcite-checkbox.tsx

```diff
@@ -41,8 +41,13 @@ export class CalciteCheckbox {
     this.calciteCheckboxChange.emit();
   }
 
-  /** The hovered state of the checkbox. */
-  @Prop({ reflect: true, mutable: true }) hovered = false;
+  /** True if the checkbox is disabled */
+  @Prop({ reflect: true }) disabled?: boolean = false;
```

Comment:
Minor: but I don't think we need to mark props as optional if they have a default value. It doesn't seem to doc them any differently unless they don't have a default value.

Reply:
Maybe cleanup for another time :)

Reply:
Agreed and yes, we can do a focused follow-up for this.